### PR TITLE
Tutorial: Guide to developing a travel application: Xata, Next.js, Tailwind CSS, & Vercel

### LINKED ISSUE #273: [Illustration request] Tutorial blog - building a travel application
Copy doc: https://docs.google.com/document/d/1nYndII6P7EU7rHPZjkt9szGKCZrXq_cu-J9jcEWWAW8/edit?pli=1#heading=h.de5xlwoszriv

Hi @miukimiu, we are planning to publish a new tutorial blog tomorrow (if possible) and I'd love your help with an illustration for /blog and the OGImage please. The copy doc is above. 

The PR I've done is broken right now so not showing a preview but hopefully the copy doc is enough for now? Let me know if you need anything else. Thank you so much.

### PR DESCRIPTION
**Pull request by CezzaineZ**
## Summary
Tutorial walking through how to create a travel application using Xata, Next.js, Tailwind CSS, & Vercel

Closes #273.

### Blog Release Checklist

- \[ ] Release on: \<06/05/24>
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

File: travel-application-development-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,1594 @@
...
+
+![Travel application landing page](/images/travel-app-guide/landing-page.png)
+
+## Before you begin
+
+### Prerequisites
+
+Before you begin, you’ll need to do the following:
+
+- Have an Xata account [(Sign up for a free)(https://app.xata.io/signin?mode=signup)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:31]:
```suggestion
- Have an Xata account [(Sign up for a free)](https://app.xata.io/signin?mode=signup)
```

File: travel-application-development-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,1594 @@
...
+- Install [Node.js(https://nodejs.org/en/download)] 18 or later
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:32]:
```suggestion
- Install [Node.js](https://nodejs.org/en/download) 18 or later
```

File: travel-application-development-guide.mdx [hunk 3]
```diff
@@ -0,0 +1,1594 @@
...
+- Create a [Vercel(https://vercel.com/)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:33]:
```suggestion
- Create a [Vercel](https://vercel.com/)
```

File: travel-application-development-guide.mdx [hunk 4]
```diff
@@ -0,0 +1,1594 @@
...
+This guide walked through the seamless integration of Xata and Next.js to build a travel-tracking application. With these technologies, you can now relive and document all the key moments from your trips. Integrating Tailwind and React Toastify helped to create a better user interface and experience.
+
+The union between Xata and Next.js helps developers build dynamic and serverless applications, whether you are an experienced developer or a novice. It is an all-encompassing way to deliver a 360º approach to full-stack CRUD applications.
+
+Happy coding!
+
+More information
+For more detailed insights, explore the references in this post below:
+
+- [Demo track trips (https://track-trip-dashboard-with-xata-next.vercel.app/)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:1588]:
```suggestion
- [Demo track trips](https://track-trip-dashboard-with-xata-next.vercel.app/)
```

File: travel-application-development-guide.mdx [hunk 5]
```diff
@@ -0,0 +1,1594 @@
...
+- [GitHub repo (https://github.com/Terieyenike/track-trip-dashboard-with-xata-next)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:1589]:
```suggestion
- [GitHub repo](https://github.com/Terieyenike/track-trip-dashboard-with-xata-next)
```

File: travel-application-development-guide.mdx [hunk 6]
```diff
@@ -0,0 +1,1594 @@
...
+- [Xata file attachments (https://xata.io/docs/sdk/file-attachments)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:1590]:
```suggestion
- [Xata file attachments](https://xata.io/docs/sdk/file-attachments)
```

File: travel-application-development-guide.mdx [hunk 7]
```diff
@@ -0,0 +1,1594 @@
...
+- [Route groups in Next.js (https://nextjs.org/docs/app/building-your-application/routing/route-groups)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:1591]:
```suggestion
- [Route groups in Next.js](https://nextjs.org/docs/app/building-your-application/routing/route-groups)
```

File: travel-application-development-guide.mdx [hunk 8]
```diff
@@ -0,0 +1,1594 @@
...
+- [Next.js with Xata (https://xata.io/docs/getting-started/nextjs)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:1592]:
```suggestion
- [Next.js with Xata](https://xata.io/docs/getting-started/nextjs)
```

File: travel-application-development-guide.mdx [hunk 9]
```diff
@@ -0,0 +1,1594 @@
...
+- [Add notifications to apps (https://github.com/fkhadra/react-toastify#readme)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:1593]:
```suggestion
- [Add notifications to apps](https://github.com/fkhadra/react-toastify#readme)
```

File: travel-application-development-guide.mdx [hunk 10]
```diff
@@ -0,0 +1,1594 @@
...
+- [Fetching, caching, revalidating (https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating)]
```
Comment (CezzaineZ) [inline travel-application-development-guide.mdx:1594]:
```suggestion
- [Fetching, caching, revalidating](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating)
```

**Review by CezzaineZ — COMMENTED** [review]
Fixing broken links

File: travel-application-development-guide.mdx (hunk 10 above)
Comment (miukimiu) [reply travel-application-development-guide.mdx:1592]: Thanks, @CezzaineZ. I had to revert the file to the initial commit where the code snippets were correct. That's why some things went back to what @alexfrancoeur already fixed. I'm currently trying to fix some errors with code snippets. But once I finish I'll fix the rest.

**Comment by miukimiu** [comment]
@alexfrancoeur the code responsible for rendering images in MDX restricts their width to a maximum of `1000px`:

https://github.com/xataio/frontend-next/blob/9358de76f3a09e204bfe1d623014960814b3850e/packages/components/src/image/ArticleImage.tsx#L21

Even if the images are wider than `1000px`, they get resized to `1000px`. So, I recommend that the images be `1000px` wide.

**Comment by CezzaineZ** [comment]
OK I'll work on resizing the images now. @alexfrancoeur what was your thoughts from Dave on....

Screenshot 2024-06-06 at 16 36 23

**Review by alexfrancoeur — APPROVED** [review]
🚢

**Pull request merged**